TestViewer/MainPage.xaml.cs

```diff
@@ -0,0 +1,84 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using System.Threading.Tasks;
+using System.Diagnostics;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.Storage;
+using Windows.Storage.Streams;
+using Windows.UI.Composition;
+using Windows.UI.Composition.Scenes;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Hosting;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+using SceneLoaderComponent;
+
+// The Blank Page item template is documented at https://go.microsoft.com/fwlink/?LinkId=402352&clcid=0x409
+
```

Comment:
Do you want the C# code reviewed? Looks like it hasn't been through review but I'm not sure if you care.

Reply:
I didn't write the C# code but if you have the bandwidth, I would appreciate a review of it as well. Thanks!

---
In reply to: [280248257](https://github.com/windows-toolkit/SceneLoader/pull/1#discussion_r280248257) [](ancestors = 280248257)

Reply:
OK... First thing is to remove the comment about the Blank Page item.

---
In reply to: [280555841](https://github.com/windows-toolkit/SceneLoader/pull/1#discussion_r280555841) [](ancestors = 280555841,280248257)